### DISCUSSION TITLE
New Source: Weaviate

### DISCUSSION THREAD

**Discussion by samos123** [airbytehq/airbyte#36106] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination?
Weaviate source and destination

* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB? 

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? This helps us understand the use case.
Allow loading data from Mongo and snowflake into Weaviate for doing semantic search with Weaviate

* How often do you want to run syncs?
Ideally stay in sync through streaming

* If this is an API source connector, which entities/endpoints do you need supported?
Weaviate is more like a database

* If the connector is for a paid service, can we name you as a mutual user when we subscribe for an account? Which company should we name? 
Not a paid service, it's OSS that's also available through a managed service.

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
Building  Airflow DAG for moving data between Weaviate and other databases

## Are you willing to submit a PR?
Yes

**Comment by erohmensing** [comment]
@samos123 looks like you added the weaviate destination, thank you! Should this issue be closed? Your description makes it look like the destination (not source) is what you were after.

[a comment by an automated account is not shown]
